utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,112 @@
+    "log_dir": "logs"
+}
+
+# ——— Prepare log file ——————————————————————————————————————————————
+os.makedirs(CONFIG["log_dir"], exist_ok=True)
+ts = datetime.now().strftime("%Y%m%d_%H%M%S")
+logfile = os.path.join(CONFIG["log_dir"], f"decrypt_json_{ts}.log")
+
+logger = logging.getLogger("decrypt_json")
+logger.setLevel(logging.INFO)
+fmt = logging.Formatter("%(asctime)s %(levelname)-8s %(message)s")
+ch = logging.StreamHandler(); ch.setFormatter(fmt); logger.addHandler(ch)
```

Flake8 (7.2.0)

[error] multiple statements on one line (semicolon)

(E702)